lib/opensusebasetest.pm

```diff
@@ -103,8 +107,8 @@ done", "binaries-with-missing-libraries.txt", {timeout => 60, noupload => 1});
         clear_console;
     }
 
-    script_run 'tar cvvJf problem_detection_logs.tar.xz *';
-    upload_logs('problem_detection_logs.tar.xz');
+    script_run 'tar cvvJf problem_detection.tar.xz *';
+    upload_logs('problem_detection.tar.xz');
```

Comment:
@foursixnine 
My idea, was to have the coredumps in the same place where the related logs are.... 

There's also no guarantee that coredumps are generated, so having an archive "core_dumps.tar.xz" and no core dumps just logs doesn't seem a good option to me.

Maybe some other name, like 'logs_and_cores.tar.xz' ?